demo/src/screens/MenuStructure.js

```diff
@@ -74,8 +74,9 @@ export const navigationData = {
         screen: 'unicorn.nativeComponents.HighlightOverlayViewScreen'
       },
       {title: 'Wheel Picker', tags: 'wheel picker', screen: 'unicorn.nativeComponents.WheelPickerViewScreen'},
-      {title: 'SafeArea Spacer', tags: 'native safe area', screen: 'unicorn.nativeComponents.SafeAreaSpacerViewScreen'}
+      {title: 'SafeArea Sapcer', tags: 'native safe area', screen: 'unicorn.nativeComponents.SafeAreaSpacerViewScreen'}
```

Comment:
I think you accidentally change the name here 
```suggestion
      {title: 'SafeArea Spacer', tags: 'native safe area', screen: 'unicorn.nativeComponents.SafeAreaSpacerViewScreen'}
```

Reply:
Fixed